Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/DeletionProcessGracePeriod/HandlerTests.cs

```diff
@@ -0,0 +1,151 @@
+﻿using Backbone.BuildingBlocks.Application.PushNotifications;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Devices.Application.Identities.Commands.DeletionProcessGracePeriod;
+using Backbone.Modules.Devices.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Devices.Application.Infrastructure.PushNotifications.DeletionProcess;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using FakeItEasy;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Application.Tests.Tests.Identities.Commands.DeletionProcessGracePeriod;
+public class HandlerTests
+{
+    [Fact]
+    public async Task No_Active_Deletion_Processes()
+    {
+        // Arrange
+        var mockIdentitiesRepository = A.Fake<IIdentitiesRepository>();
+        var mockPushNotificationSender = A.Fake<IPushNotificationSender>();
+
+        A.CallTo(() => mockIdentitiesRepository.FindAllWithApprovedDeletionProcess(A<CancellationToken>._, A<bool>._))
+            .Returns(new List<Identity>());
+
+        var handler = CreateHandler(mockIdentitiesRepository, mockPushNotificationSender);
+
+        // Act
+        await handler.Handle(new DeletionProcessGracePeriodCommand(), CancellationToken.None);
+
+        // Assert
+        A.CallTo(() => mockIdentitiesRepository.FindAllWithApprovedDeletionProcess(A<CancellationToken>._, A<bool>._))
+            .MustHaveHappenedOnceExactly();
+        A.CallTo(() => mockIdentitiesRepository.Update(A<Identity>._, A<CancellationToken>._))
+            .MustNotHaveHappened();
+        A.CallTo(() => mockPushNotificationSender.SendNotification(A<IdentityAddress>._, A<object>._, A<CancellationToken>._))
+            .MustNotHaveHappened();
+    }
+
+    [Fact]
+    public async Task Sends_First_Reminder()
+    {
+        // Arrange
+        var beginProcessDate = DateTime.Parse("2000-01-01");
+        SystemTime.Set(beginProcessDate);
```

Comment:
What's that code for?
1. You never use the `beginProcessDate`.
2. You don't care what the processe's `CreatedAt` date is.

Remove it for the other tests as well